.circleci/config.yml

```diff
@@ -35,6 +35,464 @@ commands:
+            - v1-jrs-summary-history-{{ epoch }}
+            - v1-jrs-summary-history-
+      - gcp_storage_rsync:
+          src: "gs://<< parameters.bucket_name >>/"
+          dest: "<< parameters.summary_path >>"
+
+
+  ######################################################################################################################
+  # Command Name:                     jrs_history_store
+  # Command Version:                  1.0
+  #
+  # Parameters:
+  #       bucket_name                 the name of the GCP bucket containing the JRS summary history files, mandatory
+  #       summary_path                the local path of the JRS summary history files, mandatory
+  #
+  #
+  # Description:
+  #
+  #   Saves the JRS Summary History files from the current run using a combination of CircleCI caching and GCP bucket
+  #   storage as specified by the `bucket_name` parameter. The local summary history path specified by the
+  #   `summary_path` parameter must exist.
+  #
+  #   The GCP bucket should be a dedicated storage bucket that must only contain JRS summary history files.
+  #
+  #   NOTE: The build will fail if the `gcloud` command is not available or no CRC32 system library is available.
+  #
+  # Command Requirements:
+  #   - gcloud (>= 323.0.0)
+  #
+  ######################################################################################################################
+  jrs_history_store:
+    parameters:
+      bucket_name:
+        type: string
+      summary_path:
+        type: string
+    steps:
+      - save_cache:
+          name: Saving JRS Summary History Cache
+          key: v1-jrs-summary-history-{{ epoch }}
+          paths:
+            - << parameters.summary_path >>
+          when: always
+      - gcp_storage_rsync:
+          src: "<< parameters.summary_path >>"
+          dest: "gs://<< parameters.bucket_name >>/"
+
+
+  ######################################################################################################################
+  # Command Name:                     jrs_results_store
+  # Command Version:                  1.0
+  #
+  # Parameters:
+  #       bucket_name                 the name of the GCP bucket containing the JRS regression results, mandatory
+  #       result_path                 the local path of the JRS regression result files, mandatory
+  #
+  #
+  # Description:
+  #
+  #   Saves the JRS Regression result files from the current run to the GCP bucket storage as specified by the
+  #   `bucket_name` parameter. The local result path specified by the `result_path` parameter must exist.
+  #
+  #   The GCP bucket should be a dedicated storage bucket that must only contain JRS Regression result files.
+  #
+  #   The final path used to the store the result files in the GCP bucket is as follows:
+  #       - gs://${BUCKET_NAME}/${JRS_USER}/${JRS_BRANCH}
+  #
+  #   The ${JRS_USER} variable will default to a value of `swirlds-automation` if not otherwise provided by CircleCI.
```

Comment:
Better change `swirlds-automation` to `hedera-services-automation`.

Reply:
Fixed. Thanks @ljianghedera